services/121-service/src/payments/fsp-integration/commercial-bank-ethiopia/commercial-bank-ethiopia.service.ts

```diff
@@ -230,23 +230,23 @@ export class CommercialBankEthiopiaService
         data.fieldName === FinancialServiceProviderAttributes.bankAccountNumber
       ) {
         bankAccountNumber = data.value;
-      } else if ((data.fieldName = 'debitTheIrRef')) {
-        debitTheIrRefRetry = data.value;
+      } else if ((data.fieldName = 'debitTheirRef')) {
+        debitTheirRefRetry = data.value;
       }
     });
 
     return {
       debitAmount: payment.transactionAmount,
-      debitTheIrRef:
-        debitTheIrRefRetry ||
+      debitTheirRef:
+        debitTheirRefRetry ||
         `${formatDateYYMMDD(new Date())}${this.generateRandomNumerics(10)}`,
-      creditTheIrRef: program.ngo,
-      creditAcctNo: bankAccountNumber,
-      creditCurrency: program.currency,
-      remitterName:
+      creditTheirRef:
```

Comment:
@elwinschmitz you switched here the values of 'creditTheirRef' and 'remitterName' around right? Is that intentional, and are you sure of that?

Reply:
ah wait sorry I only now see the other task in Review which explains this. Makes sense, given the documentation! And at least clear, that both are not vital properties that break if switched.